### PR TITLE
Add option to make neurons join the community fund

### PR DESCRIPTION
**Pull request by hpeebles**

### CODE REVIEW

File: frontend/ts/src/canisters/governance/rawService.ts [hunk 1]
```diff
@@ -284,6 +291,9 @@ export interface Tally {
   'total' : bigint,
   'timestamp_seconds' : bigint,
 }
+export interface UpdateNodeProvider {
```
Comment (dralves) [inline frontend/ts/src/canisters/governance/rawService.ts:294]: does this update node provider stuff have to go in with the community fund stuff?

File: frontend/ts/src/canisters/governance/canister_certified.did.js [hunk 2]
```diff
@@ -499,11 +578,12 @@ export const init = ({ IDL }) => {
...
   });
   Proposal.fill(
-    IDL.Record({
-      'url' : IDL.Text,
-      'action' : IDL.Opt(Action),
-      'summary' : IDL.Text,
-    })
+      IDL.Record({
+        'url' : IDL.Text,
+        'title' : IDL.Opt(IDL.Text),
```
Comment (dralves) [inline frontend/ts/src/canisters/governance/canister_certified.did.js:583]: this also includes title related changes?

File: frontend/ts/src/canisters/governance/rawService.ts (hunk 1 above)
Comment (hpeebles) [reply frontend/ts/src/canisters/governance/rawService.ts:294]: No, but this is autogenerated from the candid file.

File: frontend/ts/src/canisters/governance/canister_certified.did.js (hunk 2 above)
Comment (hpeebles) [reply frontend/ts/src/canisters/governance/canister_certified.did.js:583]: No, but this is autogenerated from the candid file.

**Review by dralves — APPROVED** [review]

**Comment by dralves** [comment]
@hpeebles cleared to me that the changes that I pointed out are automatically generated.

**Pull request merged**